generic_config_updater/gcu_table_modification_validators.conf.json

```diff
@@ -0,0 +1,39 @@
+{
+    "README": [
+        "table_modification_validators provides, module & method name as ",
+        "    <module name>.<method name>",
+        "NOTE: module name could have '.'",
+        " ",
```

Comment:
Do we need testcase for this file? Like tests in `service_validator_test.py`?

Reply:
I added additional unit testing for the new RDMA-specific validator as suggested offline